Card/src/main/java/com/braintreepayments/api/card/CardClient.kt

```diff
@@ -0,0 +1,134 @@
+package com.braintreepayments.api.card
+
+import android.content.Context
+import androidx.annotation.VisibleForTesting
+import com.braintreepayments.api.card.CardNonce.Companion.fromJSON
+import com.braintreepayments.api.core.ApiClient
+import com.braintreepayments.api.core.BraintreeClient
+import com.braintreepayments.api.core.BraintreeException
+import com.braintreepayments.api.core.Configuration
+import com.braintreepayments.api.core.GraphQLConstants
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Used to tokenize credit or debit cards using a [Card]. For more information see the
+ * [documentation](https://developer.paypal.com/braintree/docs/guides/credit-cards/overview)
+ */
+class CardClient @VisibleForTesting internal constructor(
+    private val braintreeClient: BraintreeClient,
+    private val apiClient: ApiClient
+) {
+
+    /**
+     * Initializes a new [CardClient] instance
+     *
+     * @param context an Android Context
+     * @param authorization a Tokenization Key or Client Token used to authenticate
+     */
+    constructor(context: Context, authorization: String) : this(
+        BraintreeClient(
+            context,
+            authorization
+        )
+    )
+
+    @VisibleForTesting
+    internal constructor(braintreeClient: BraintreeClient) : this(
```

Comment:
Could we consolidate this constructor by setting a default value for `apiClient` in the primary constructor?